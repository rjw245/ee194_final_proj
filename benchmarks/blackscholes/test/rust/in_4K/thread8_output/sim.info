{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread8_output',
             '-n',
             '8',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../rust/bs',
             '8',
             '../../inputs/in_4K.txt'],
 'git_revision': '',
 'host': 'dell26',
 'pin_version': 71313,
 'rusage': (843.015842,
            4.46732,
            141492,
            0,
            0,
            0,
            453246,
            4,
            0,
            2168,
            12464,
            0,
            0,
            0,
            138799,
            805200),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=8 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/rust/thread8_output --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../rust/bs 8 ../../inputs/in_4K.txt'],
 't_elapsed': 1183.3457188606262,
 't_start': 1460948560.157334,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 4083752960L}
