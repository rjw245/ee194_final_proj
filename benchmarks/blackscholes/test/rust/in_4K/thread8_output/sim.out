                                     | Core 0     | Core 1     | Core 2     | Core 3     | Core 4     | Core 5     | Core 6     | Core 7    
  Instructions                       |  276502344 |    8390656 |     123800 |      62876 |      63325 |      63401 |      64051 |      62775
  Cycles                             |  167593034 |  167593034 |  167593034 |  167593034 |  167593034 |  167593034 |  167593034 |  167593034
  IPC                                |       1.65 |       0.05 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00
  Time (ns)                          |   63004900 |   63004900 |   63004900 |   63004900 |   63004900 |   63004900 |   63004900 |   63004900
  Idle time (ns)                     |    2523676 |   60454833 |   62963575 |   62983238 |   62980693 |   62984621 |   62984006 |   62979815
  Idle time (%)                      |       4.0% |      96.0% |      99.9% |     100.0% |     100.0% |     100.0% |     100.0% |     100.0%
Branch predictor stats               |            |            |            |            |            |            |            |           
  num correct                        |    9788682 |     303569 |      11333 |       5699 |       5759 |       5769 |       5827 |       5695
  num incorrect                      |     437411 |       5829 |        946 |        592 |        593 |        595 |        615 |        587
  misprediction rate                 |      4.28% |      1.88% |      7.70% |      9.41% |      9.34% |      9.35% |      9.55% |      9.34%
  mpki                               |       1.58 |       0.69 |       7.64 |       9.42 |       9.36 |       9.38 |       9.60 |       9.35
TLB Summary                          |            |            |            |            |            |            |            |           
  I-TLB                              |            |            |            |            |            |            |            |           
    num accesses                     |   56317554 |    1411856 |      15922 |       8286 |       8345 |       8365 |       8462 |       8248
    num misses                       |        211 |         83 |         77 |         69 |         69 |         69 |         69 |         69
    miss rate                        |      0.00% |      0.01% |      0.48% |      0.83% |      0.83% |      0.82% |      0.82% |      0.84%
    mpki                             |       0.00 |       0.01 |       0.62 |       1.10 |       1.09 |       1.09 |       1.08 |       1.10
  D-TLB                              |            |            |            |            |            |            |            |           
    num accesses                     |  184060825 |    4809848 |      44866 |      22434 |      22521 |      22539 |      22742 |      22414
    num misses                       |       4555 |        706 |        148 |         63 |         68 |         65 |         70 |         70
    miss rate                        |      0.00% |      0.01% |      0.33% |      0.28% |      0.30% |      0.29% |      0.31% |      0.31%
    mpki                             |       0.02 |       0.08 |       1.20 |       1.00 |       1.07 |       1.03 |       1.09 |       1.12
  L2 TLB                             |            |            |            |            |            |            |            |           
    num accesses                     |       4766 |        789 |        225 |        132 |        137 |        134 |        139 |        139
    num misses                       |        424 |        217 |        173 |        126 |        126 |        128 |        128 |        130
    miss rate                        |      8.90% |     27.50% |     76.89% |     95.45% |     91.97% |     95.52% |     92.09% |     93.53%
    mpki                             |       0.00 |       0.03 |       1.40 |       2.00 |       1.99 |       2.02 |       2.00 |       2.07
Cache Summary                        |            |            |            |            |            |            |            |           
  Cache L1-I                         |            |            |            |            |            |            |            |           
    num cache accesses               |   56317554 |    1411856 |      15922 |       8286 |       8345 |       8365 |       8462 |       8248
    num cache misses                 |    1290217 |       1078 |       1412 |        885 |        884 |        882 |        898 |        889
    miss rate                        |      2.29% |      0.08% |      8.87% |     10.68% |     10.59% |     10.54% |     10.61% |     10.78%
    mpki                             |       4.67 |       0.13 |      11.41 |      14.08 |      13.96 |      13.91 |      14.02 |      14.16
  Cache L1-D                         |            |            |            |            |            |            |            |           
    num cache accesses               |  184061351 |    4810098 |      45390 |      22731 |      22834 |      22860 |      23090 |      22709
    num cache misses                 |      15811 |       3557 |       1631 |        624 |        632 |        617 |        617 |        607
    miss rate                        |      0.01% |      0.07% |      3.59% |      2.75% |      2.77% |      2.70% |      2.67% |      2.67%
    mpki                             |       0.06 |       0.42 |      13.17 |       9.92 |       9.98 |       9.73 |       9.63 |       9.67
  Cache L2                           |            |            |            |            |            |            |            |           
    num cache accesses               |    1306212 |       4655 |       3072 |       1514 |       1565 |       1519 |       1529 |       1527
    num cache misses                 |      12679 |       4081 |       2213 |       1440 |       1454 |       1436 |       1450 |       1425
    miss rate                        |      0.97% |     87.67% |     72.04% |     95.11% |     92.91% |     94.54% |     94.83% |     93.32%
    mpki                             |       0.05 |       0.49 |      17.88 |      22.90 |      22.96 |      22.65 |      22.64 |      22.70
  Cache L3                           |            |            |            |            |            |            |            |           
    num cache accesses               |      12880 |       4103 |       2243 |       1445 |       1503 |       1456 |       1464 |       1456
    num cache misses                 |      11090 |        818 |       1117 |        400 |        960 |        405 |        399 |        911
    miss rate                        |     86.10% |     19.94% |     49.80% |     27.68% |     63.87% |     27.82% |     27.25% |     62.57%
    mpki                             |       0.04 |       0.10 |       9.02 |       6.36 |      15.16 |       6.39 |       6.23 |      14.51
DRAM summary                         |            |            |            |            |            |            |            |           
  num dram accesses                  |       7338 |          0 |          0 |          0 |       7353 |          0 |          0 |          0
  average dram access latency (ns)   |      92.34 |        inf |        inf |        inf |      93.01 |        inf |        inf |        inf
  average dram queueing delay        |      38.91 |       0.00 |       0.00 |       0.00 |      39.58 |       0.00 |       0.00 |       0.00
  average dram bandwidth utilization |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00%
Coherency Traffic                    |            |            |            |            |            |            |            |           
  num loads from dram                |       3253 |        307 |         53 |         25 |         15 |         24 |         24 |         18
  num loads from dram cache          |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  num loads from remote cache        |         32 |         23 |         44 |         12 |        153 |         31 |         25 |         86
